dali/kernels/imgproc/flip_cpu.h

```diff
@@ -46,39 +46,48 @@ void OcvFlip(Type *output, const Type *input,
     flip_flag = 0;
   // coerce data to uint8_t - flip won't look at the type anyway
   auto ocv_type = GetOcvType<uint8_t>(channels * sizeof(Type));
-  size_t layer_size = height * width * channels;
-  for (size_t layer = 0; layer < layers; ++layer) {
-    auto output_layer = flip_z ? layers - layer - 1 : layer;
-    auto input_mat = CreateMatFromPtr(height, width, ocv_type, input + layer * layer_size);
+  size_t plane_size = height * width * channels;
+  for (size_t plane = 0; plane < depth; ++plane) {
+    auto output_plane = flip_z ? depth - plane - 1 : plane;
+    auto input_mat = CreateMatFromPtr(height, width, ocv_type, input + plane * plane_size);
     auto output_mat = CreateMatFromPtr(height, width, ocv_type,
-                                       output + output_layer * layer_size);
+                                       output + output_plane * plane_size);
     cv::flip(input_mat, output_mat, flip_flag);
   }
 }
 
 template <typename Type>
-void FlipZAxis(Type *output, const Type *input, size_t layers, size_t height, size_t width,
+void FlipZAxis(Type *output, const Type *input, size_t depth, size_t height, size_t width,
                       size_t channels, bool flip_z) {
   if (!flip_z) {
-    std::copy(input, input + layers * height * width * channels, output);
+    std::copy(input, input + depth * height * width * channels, output);
     return;
   }
-  size_t layer_size = height * width * channels;
-  for (size_t layer = 0; layer < layers; ++layer) {
-    auto out_layer = layers - layer - 1;
-    std::copy(input + layer * layer_size, input + (layer + 1) * layer_size,
-              output + out_layer * layer_size);
+  size_t plane_size = height * width * channels;
+  for (size_t plane = 0; plane < depth; ++plane) {
+    auto out_plane = depth - plane - 1;
+    std::copy(input + plane * plane_size, input + (plane + 1) * plane_size,
+              output + out_plane * plane_size);
   }
 }
 
 template <typename Type>
 void FlipImpl(Type *output, const Type *input,
-                       size_t layers, size_t height, size_t width,
-                       size_t channels, bool flip_z, bool flip_y, bool flip_x) {
+              size_t seq_length, size_t depth, size_t height, size_t width,
+              size_t channels, bool flip_z, bool flip_y, bool flip_x) {
+  auto frame_size = depth * height * width * channels;
```

Comment:
So we want to support sequences of 3D images?

Reply:
It doesn't cost much and if it supports sequences and 3D then why not 3D sequences

Reply:
Can we perhaps reverse sequences with this operator? Would that be useful?